### ISSUE TITLE
Spin component should be accessible by default

### ISSUE THREAD

**Issue by frankandrobot** [ant-design/ant-design#34377]
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
Makes the spin component accessible out of the box.

I'm currently putting an ant-based component under test with react-testing-library. I was unable to find a suitable selector for the Spin component with the default settings. The recommendation is to add aria-live and aria-busy to the Spin component *by default*. 

```
    <Spin spinning={isLoading} aria-live="polite" aria-busy={isLoading}>
```

With the above properties, the Spin component should play well with screen readers. And so this behavior should be *default* and not add on. 

### What does the proposed API look like?
See above.

**Comment by afc163** [comment]
Fixed in https://github.com/ant-design/ant-design/pull/34408
